contracts/pm/mixins/MixinReserve.sol

```diff
@@ -11,67 +11,61 @@ contract MixinReserve is MContractRegistry, MReserve {
     }
 
-    struct ReserveManager {
-        uint256 reserveNonce;                                                 // Current reserve ID
-        Reserve reserve;                                                      // Storage pointer to a reserve
-        mapping (uint256 => mapping (address => uint256)) claimedPerReserve;  // Mapping of reserve ID => claimant address => amount claimed for reserve ID
-    }
-
-    // Mapping of address => managed reserve for an address
-    mapping (address => ReserveManager) internal reserveManagers;
-
-    // Number of rounds before a frozen reserve thaws
-    uint256 public freezePeriod;
+    // Mapping of address => reserve
+    mapping (address => Reserve) internal reserves;
 
     /**
      * @dev Returns info about a reserve
      * @param _reserveHolder Address of reserve holder
      * @return Info about the reserve for `_reserveHolder`
      */
     function getReserveInfo(address _reserveHolder) public view returns (ReserveInfo memory info) {
-        Reserve storage reserve = reserveManagers[_reserveHolder].reserve;
-
         info.fundsRemaining = remainingReserve(_reserveHolder);
-        info.state = reserveState(_reserveHolder);
-        info.thawRound = reserve.freezeRound == 0 ? 0 : reserve.freezeRound.add(freezePeriod);
+        info.claimedInCurrentRound = reserves[_reserveHolder].claimedForRound[roundsManager().currentRound()];
     }
 
     /**
-     * @dev Returns the amount of funds claimable by a claimant from a reserve
+     * @dev Returns the amount of funds claimable by a claimant from a reserve in the current round
      * @param _reserveHolder Address of reserve holder
      * @param _claimant Address of claimant
-     * @return Amount of funds claimable by `_claimant` from the reserve for `_reserveHolder`
+     * @return Amount of funds claimable by `_claimant` from the reserve for `_reserveHolder` in the current round
      */
     function claimableReserve(address _reserveHolder, address _claimant) public view returns (uint256) {
-        ReserveManager storage manager = reserveManagers[_reserveHolder];
-        Reserve storage reserve = manager.reserve;
-        if (reserveState(_reserveHolder) == ReserveState.NotFrozen) {
-            // TODO: Check if claimant is registered in current round
-            uint256 poolSize = bondingManager().getTranscoderPoolSize();
-            return poolSize > 0 ? reserve.fundsAdded.div(poolSize) : 0;
-        } else {
-            // TODO: Check if claimant is registered during the freezeRound
-            return reserve.recipientsInFreezeRound > 0 ?
-                reserve.fundsAdded.div(reserve.recipientsInFreezeRound).sub(manager.claimedPerReserve[manager.reserveNonce][_claimant]) :
-                0;
+        Reserve storage reserve = reserves[_reserveHolder];
+
+        uint256 currentRound = roundsManager().currentRound();
+
+        // TODO: Check if claimant is active in the current round
```

Comment:
I assume we aren't implementing the active check yet because the function signature changes in #318 ?

Reply:
Yep I wanted to wait for the BondingManager changes to land first.